### ISSUE TITLE
Create a FacebookAuthorName component

### ISSUE THREAD

**Issue by andizer** [Yoast/yoast-components#808]
The `FacebookAuthorName` will be shown as a div:
![yoast_-_home](https://user-images.githubusercontent.com/325040/51387794-aeb0af00-1b27-11e9-90b8-0ff12d04b697.png)

This component can be combined with the `FacebookSiteName` component (See: #807), because of most styling is equal.

When the name is present it should start with a separator (`|`). The name itself should be designed as a link.

## Styling
The textual styling for this will be:
```css
    color: #606770;
    flex-shrink: 0;
    font-size: 12px;
    line-height: 16px;
    overflow: hidden;
    padding: 0;
    text-overflow: ellipsis;
    text-transform: uppercase;
    white-space: nowrap;
}
```

The styling for the name (link) will be:
```css
    color: #3b5998;
    position: relative;
    text-decoration: none;
    cursor: pointer
    z-index: 5;
```

Parent

**Comment by IreneStr** [comment]
@luckickken On Facebook, the author name is a link to the person's facebook page (set for the user in the WP admin). Because it's a preview, I presume we don't want the link to be an actual link (please correct me if I'm wrong).  Does that also mean that we don't want `cursor: pointer` on the author name?

**Comment by luckickken** [comment]
@IreneStr I agree. I think the link should just be decorative. `cursors: auto` should work in this case.
